02_configure_host.sh

```diff
@@ -169,7 +166,7 @@ method=manual
 addr-gen-mode=eui64
 method=disabled
 EOF
-     	  fi
+     fi
```

Comment:
Something wrong with the indents on this or the next `fi`, they're on same level now.